python/pipeline/reso.py

```diff
@@ -514,10 +511,6 @@ def _make_tuples(self, key):
             self.notify({**key, 'slice': slice_id + 1}, scan) # once per slice
 
     def notify(self, key, scan):
-        # --  notification
-        import matplotlib
-        matplotlib.rcParams['backend'] = 'Agg'
-        import matplotlib.pyplot as plt
```

Comment:
It is imported at the very top of reso.py (because it is used in a lot of places). I did test it, I'll check again now while I try to rescale cnmf traces to the right range(I think I know how, so don't merge this yet)